src/main/resources/schema.sql

```diff
@@ -45,22 +68,6 @@ CREATE TABLE program (
     UNIQUE (name, organization_id)
 );
 
-CREATE TABLE program_area (
```

Comment:
I reordered this to be easier to see that the drops & creates were in the proper order.